File: src/dk/eightyplus/Painter/utilities/SaveLoad.java
package dk.eightyplus.Painter.utilities;

import android.content.Context;

import java.io.DataInputStream;
import java.io.DataOutputStream;
import java.io.IOException;

/**
 * User: fries
 * Date: 18/03/14
 * Time: 16.59
 */
public interface SaveLoad {
  public void save(final Context context, final DataOutputStream outputStream) throws IOException;
  public void load(final Context context, final DataInputStream inputStream) throws IOException, ClassNotFoundException;
}

File: app/src/main/java/dk/eightyplus/annoto/utilities/SaveLoad.kt
package dk.eightyplus.annoto.utilities

import android.content.Context

import java.io.DataInputStream
import java.io.DataOutputStream
import java.io.IOException

/**
 * User: fries
 * Date: 18/03/14
 * Time: 16.59
 */
interface SaveLoad {
    @Throws(IOException::class)
    fun save(context: Context, outputStream: DataOutputStream)

    @Throws(IOException::class, ClassNotFoundException::class)
    fun load(context: Context, inputStream: DataInputStream)
}
